js_modules/dagit/packages/core/src/app/FallthroughRoot.tsx

```diff
@@ -49,21 +52,28 @@ const FinalRedirectOrLoadingRoot = () => {
     return <Redirect to="/workspace" />;
   }
 
-  const reposWithAJob = allRepos.filter((r) => r.repository.pipelines.length > 0);
+  const reposWithVisibleJobs = allRepos.filter((r) => getVisibleJobs(r).length > 0);
```

Comment:
Could use something like this for the filter, since you're just checking `length > 0`:

```
const hasVisibleJobs = r.repository.pipelines.some((j) => !isHiddenAssetGroupJob(j.name));

// then
const reposWithVisibleJobs = allRepos.filter(hasVisibleJobs);
```

Reply:
Ahh this makes sense! let me see what I can do 🙌